### DISCUSSION TITLE
Prerelease coming weekend?

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1186] [Team Posts]
Steady strides kybos, love it!
Most things seem to work fine, saw you added most (all?) of the functionality from the transport into toolbars too, very cool. 
...Though now that I think about it, I did start a document with some noted incidents, ahem, I'll try to make tickets as soon as I remember.

Regardless, what do you guys think if I try to make a prerelease this coming weekend? I think should be able to wrap that up..

**Comment by kybos** [comment]
It's ok with me. I had started something bigger (a partial standardization of the mixer strip), lots of detail work, but it's almost finished and will be ready by the weekend.

What will be the next release number? The recent discussion about that didn't quite come to any conclusion, IIRC...

> saw you added most (all?) of the functionality from the transport into toolbars too

Yes, everything is in the toolbars now. Also the (optional) display of current position in ticks/frames, which was only available in the big time window before.

**Comment by spamatica** [comment]
Alright, cool, more Christmas presents :D

I'm very much pro going for 4.0 and changing the source tree to 'src'.
Some weeks ago I was trying to find the origin of some file in our git tree and I discovered that the history didn't continue all the way, so I didn't find the original commit.
Anyway, I see this as a clue that the source of Tim's problems with maintaining fix-branches was because the branches didn't correlate to an existing branch in git and therefore were impossible to update through git.

If we change the folder name now this will be just another commit and the file histories will still be intact.
Of course there may still be issues with updating the branches (there always is) but the renaming should not be a big problem.

But if this feels like a big decision we don't necessarily have to change the sources for this pre release.

**Comment by kybos** [comment]
For my part, I fully agree.
